assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -206,10 +296,46 @@ const baseSelectors = {
 	 * @param {Object} state Data store's state.
 	 * @return {(Object|undefined)} Modules available on the site.
 	 */
-	getModules( state ) {
+	getModules: createRegistrySelector( ( select ) => ( state ) => {
 		const { modules } = state;
-
+		const registryKey = select( STORE_NAME ).getModuleRegistryKey();
+		if ( undefined !== modules ) {
```

Comment:
Nitpick, but this might be a bit easier-to-read/follow if the conditional were the "early return". As-is, we `return modules;` after this `if` block, but it's by-definition `undefined`. Changing this section to be:

```js
if ( undefined === modules ) {
  return undefined;
}

// Continue with mappedModules code below...
```